src/Core/ConfigurationParser.re

```diff
@@ -281,6 +281,22 @@ let configurationParsers: list(configurationTuple) = [
     "editor.zenMode.singleFile",
     (s, v) => {...s, zenModeSingleFile: parseBool(v)},
   ),
+  (
+    "syntax.eagerMaxLines",
+    (s, v) => {
```

Comment:
Not a fan of `s` and `v` here. Apparently `s` is somehow short for `configuration` and `v` is short for `json`. A good example of too short variable names becoming wildly misleading because they eventually lose their meaning.

Reply:
Updated `s` -> `config` and `v` -> `json` to be more understandable